config/locales/en.yml

```diff
@@ -1187,69 +1187,183 @@ en:
-              q: What if I don’t have a bank account?
-            no_income:
-              a_html: You need to enter your payment information on the IRS %{irs_enter_payment_info_link} website to receive your payment.
-              enter_payment_info_website_title: "“Non-Filers: Enter Payment Info Here”"
-              q: What if I have no income?
-            non_filer:
-              a: If you aren’t required to file taxes but you have any earned income we recommend filing your taxes through GetYourRefund as you may be eligible for additional tax credits! If you’re required to file taxes, but didn’t file in 2018 or 2019, you can still file with GetYourRefund.
-              q: What if I haven’t filed federal taxes?
-            other_benefits:
-              a_html: You should get the payment automatically and do not need to fill out a form. If you would like to change your payment details, visit the IRS %{irs_get_my_payment_link} website.
-              q: What if I am on SSI, SSDI, Railroad Retirement Benefits, C&P?
-            payment_amount:
-              a_html: The IRS has a %{irs_eip_calculator_link} website to help you calculate how much you will receive.
-              calculate_eip_website_title: "“How do I calculate my EIP”"
-              q: How much will I receive?
-        header: Get your Stimulus Check
+            items:
+            - title: How do I get my Economic Impact Payment check?
+              content_html: >-
+                If the IRS has not sent your first or second round stimulus payment yet,
+                or if you received the wrong amount for either payment,
+                you will need to file a 2020 tax return in order to receive it.
+            - title: How do I determine if the IRS has sent my stimulus check?
+              content_html: >-
+                If you are not sure if the IRS has sent your payment(s) or want to check the status, you can search at the IRS
+                <a
+                  href="https://www.irs.gov/coronavirus/get-my-payment"
+                  rel="nofollow"
+                  target="_blank"
+                  data-track-click="stimulus-irs-get-my-payment"
+                  data-track-attribute-position="stimulus-how-faq2">
+                Get My Payment</a>
+                website.
+            - title: What if there are issues with my bank account information?
+              content_html: >-
+                If you provided bank account information earlier to the IRS that has since changed
+                or do not recognize the number in their system, please see the
+                <a href="https://www.irs.gov/newsroom/irs-statement-update-on-economic-impact-payments"
+                  rel="nofollow"
```

Comment:
Consider `rel="nofollow noopener"` here too

Also there's no real need for the `nofollow` IMHO! But it's fine to keep it.